Test_edge_creation

```diff
@@ -0,0 +1,126 @@
+var assert = require('assert');
```

Comment:
Yes.

Its my first time writing a test and doing a pull request

On Wed, Jul 8, 2015 at 2:41 PM, Dário <notifications@github.com> wrote:

> In Test_edge_creation
> <https://github.com/appscot/sails-orientdb/pull/119#discussion_r34147800>:
>
> > @@ -0,0 +1,126 @@
> > +var assert = require('assert');
>
> Was this file added by accident?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/appscot/sails-orientdb/pull/119/files#r34147800>.
>